style.css

```diff
@@ -159,3 +159,323 @@
   right: 0;
   top: 87.09%;
 }
+
+.work {
+  background-color: rgb(229 229 229);
```

Comment:
[ ] kindly fix the` background color` for the work section. the one used in the Figma design is white.🤓

![design ta3hm](https://user-images.githubusercontent.com/62156607/198575575-4acf7873-6ee8-4474-93c3-d8bf8831e8b5.PNG)